src/main/default/classes/BaseTriggerTest.cls

```diff
@@ -0,0 +1,17 @@
+@isTest
+public class BaseTriggerTest {
```

Comment:
this should be in the `support` package directory since it doesn't need to be included in release

Reply:
@ralphcallaway 

There are going to have to be tests in the main folder (at the very least to just get code coverage).  Obviously not testing actual DML, but we will need some implementing classes to get coverage

Reply:
so i was able to upload a package version without any tests in the package folder ...

secondarily the test is referencing a class in the support folder, so if this was released it wouldn't compile ...

Reply:
To a production org? Its should have the same coverage requirements as other unmanaged code.

I guess for any existing (upgraded) orgs, test coverage for come organically... 

Although it could end up with low coverage if the org isn't using some of the features.  

I think we'd want to deploy it with some coverage of the "control" features

Reply:
maybe I'm wrong....

https://developer.salesforce.com/docs/atlas.en-us.sfdx_dev.meta/sfdx_dev/sfdx_dev_unlocked_pkg_code_coverage.htm